db/schema.rb

```diff
@@ -77,11 +79,12 @@
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
     t.string "title", null: false
-    t.text "content", null: false
+    t.text "ect_summary", null: false
```

Comment:
How come ECT summary is the only one with null false in both modules and lessons?

Reply:
Ehh, it's the only one that is currently guaranteed to be there.

The others will _eventually_ be not-null, but not yet.